app/assets/javascripts/darkswarm/directives/country_selector.js.coffee

```diff
@@ -0,0 +1,11 @@
+Darkswarm.directive "countrySelector", () ->
```

Comment:
@mkllnk I've tidied it up a bit, but I still can't get it to work without the directive...